modules/ossm-vs-istio.adoc

```diff
@@ -89,3 +89,8 @@ spec:
 = The Istio Container Network Interface (CNI) plug-in
 
 {ProductName} includes CNI plug-in, which provides you with an alternate way to configure application pod networking. The CNI plug-in replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to Security Context Constraints (SCCs) with elevated privileges.
+
+= Envoy, Secret Discovery Service, and Certificates
+
+* {ProductName} does not support QUIC-based services. 
```

Comment:
QUIC?

Reply:
There's a resolved conversation about this. https://github.com/openshift/openshift-docs/pull/20952#discussion_r404878937

I don't think we need to do a whole lot of explaining about what this is if we're going to say this is not supported and that it's a difference between istio and OSSM. 

If we want to remove the whole item, I think that's the other option.

Reply:
I mostly just wanted the acronym spelled out.

Reply:
I just checked. There was a google project name, but the team pointed out to me that QUIC is a name, not an acronym. https://tools.ietf.org/html/draft-ietf-quic-transport-22#section-1.2